includes/elementor-functions.php

```diff
@@ -0,0 +1,219 @@
+/**
+ * Helper function to show/hide elements
+ *
+ * This works with switches, if the setting ID that has been passed is toggled on, we'll return show, otherwise we'll return hide
+ *
+ * @param  string $setting_id
+ * @return string|array same as the Elementor internal function does.
+ */
+function hello_show_or_hide( $setting_id ) {
+	return ( 'yes' === hello_elementor_get_setting( $setting_id ) ? 'show' : 'hide' );
+}
+
+/**
+ * Helper function to translate the header layout setting into a class name.
+ *
+ * @return string
+ */
+function hello_get_header_layout_class() {
+	$layout_classes = [];
+
+	$header_layout = hello_elementor_get_setting( 'hello_header_layout' );
+	if ( 'inverted' === $header_layout ) {
+		$layout_classes[] = 'header-inverted';
+	} elseif ( 'stacked' === $header_layout ) {
+		$layout_classes[] = 'header-stacked';
+	}
+
+	$header_width = hello_elementor_get_setting( 'hello_header_width' );
+	if ( 'full-width' === $header_width ) {
+		$layout_classes[] = 'header-full-width';
+	}
+
+	$header_menu_dropdown = hello_elementor_get_setting( 'hello_header_menu_dropdown' );
+	if ( 'tablet' === $header_menu_dropdown ) {
+		$layout_classes[] = 'menu-dropdown-tablet';
+	} elseif ( 'mobile' === $header_menu_dropdown ) {
+		$layout_classes[] = 'menu-dropdown-mobile';
+	} elseif ( 'none' === $header_menu_dropdown ) {
+		$layout_classes[] = 'menu-dropdown-none';
+	}
+
+	$hello_header_menu_layout = hello_elementor_get_setting( 'hello_header_menu_layout' );
+	if ( 'dropdown' === $hello_header_menu_layout ) {
+		$layout_classes[] = 'menu-layout-dropdown';
+	}
+
+	return implode( ' ', $layout_classes );
+}
+
+/**
+ * Helper function to translate the footer layout setting into a class name.
+ *
+ * @return string
+ */
+function hello_get_footer_layout_class() {
+	$footer_layout = hello_elementor_get_setting( 'hello_footer_layout' );
+
+	$layout_classes = [];
+
+	if ( 'inverted' === $footer_layout ) {
+		$layout_classes[] = 'footer-inverted';
+	} elseif ( 'stacked' === $footer_layout ) {
+		$layout_classes[] = 'footer-stacked';
+	}
+
+	$footer_width = hello_elementor_get_setting( 'hello_footer_width' );
+
+	if ( 'full-width' === $footer_width ) {
+		$layout_classes[] = 'footer-full-width';
+	}
+
+	if ( hello_elementor_get_setting( 'hello_footer_copyright_display' ) && '' !== hello_elementor_get_setting( 'hello_footer_copyright_text' ) ) {
+		$layout_classes[] = 'footer-has-copyright';
+	}
+
+	return implode( ' ', $layout_classes );
+}
+
+add_action( 'elementor/editor/after_enqueue_scripts', function() {
+	$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+	wp_enqueue_script(
+		'hello-theme-editor',
+		get_template_directory_uri() . '/assets/js/hello-editor' . $suffix . '.js',
+		[ 'jquery', 'elementor-editor' ],
+		HELLO_ELEMENTOR_VERSION,
+		true
+	);
+
+	wp_enqueue_style(
+		'hello-elementor-customizer',
+		get_template_directory_uri() . '/elementor-customizer' . $suffix . '.css',
+		[],
+		HELLO_ELEMENTOR_VERSION
+	);
```

Comment:
It looks like the customizer CSS is loaded in the Elementor Editor?

Reply:
@StuartOB check if you can conditionally load this, and if there is any CSS inside the Customizer CSS file that we use in the Elementor Settings panel, we must split it out.